File: data/leetCode Solutions/Q1_twoSum/Q1_twoSum.cpp
#include <bits/stdc++.h>

using namespace std;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n = nums.size();
        vector<int> ans(2);

        for (int i = 0; i < n;i++){
            for (int j = i + 1; j < n;j++){
                if(nums[i] + nums[j] == target){
                    ans[0] = i;
                    ans[1] = j;
                }
            }
        }
        return ans;
    }
};
// priya made chnges

File: data/leetCode Solutions/Q1_twoSum/Q1_twoSum.js
var twoSum = function (nums, target) {
	let arr = [];
	for (let i = 0; i < nums.length; i++){
		for (let j = i + 1; j < nums.length; j++){
			if (nums[i] + nums[j] === target) {
				arr.push(i);
				arr.push(j);
			}
		}
	}

	return arr;
};